rfcs/0013-deprecate-currency-factory.md

```diff
@@ -0,0 +1,145 @@
+# Depreciation of Currency Factory - Assessment RFC
+
+This assessment aims to gather all relevant details and consequences of 
+depreciating Currency Factory and its replacements.
+
+## Reasoning
+
+There are several reasons we stand to benefit from replacing Currency Factory.
+These reasons are detailed below:
+
+### Currency Factory Ranges are Confusing
+  
+The Currency Factory ranges are a confusing abstraction that we don't 
+stand to gain from. We really only have two types of currencies native 
+(mintable) and non-native (external). However, we currently have many more 
+ranges that enforce artificial limitations on various abstractions of these two 
+currency types. This complicates the currency creation process and leads to 
+confusion while providing no benefit to us.
+
+### No Permissionless Asset Creation
+
+> We do not have permissionless asset (non-payable/no sufficient) - it is 
+impossible to create no marketable assets (which is needed for 
+governance/airdrops).
+
+### Currency Factory is Already Losing Functionality
+
+Through RFCs 0009 and 0010, we decided to move away from storing ED in Currency 
+Factory. After the removal of ED storage, Currency Factory was left with two 
+uses: reserving asset IDs and storing asset metadata. 
+
+As discussed, Currency Factory overcomplicates the asset ID reservation system. 
+As for asset metadata, we don't have standards for formatting this nor do we 
+utilize it. Assuming we replace the reservation system, Currency Factory will be 
+left without value.
+
+## Solution Proposals
```

Comment:
Going directly to solutions without giving any concrete requirements fulfilled by this solution based on the provided reasoning seems odd to me. That means the readers can not actually assess whether a solution is both necessary and sufficient.